Packs/CommonPlaybooks/Playbooks/playbook-Command-Line_Analysis.yml

```diff
@@ -1566,6 +1592,35 @@ outputs:
 - contextPath: CommandlineVerdict
   description: The command line verdict
   type: unknown
+- contextPath: IP
+  description: The IP object.
+  type: unknown
+- contextPath: URL
+  description: The URL object.
+  type: uknown
+- contextPath: File
+  description: The file object.
+  type: unknown
+- contextPath: Domain
+  description: The domain object.
+  type: unknown
 tests:
-- No tests (auto formatted)
+- Calculate Severity - Generic v2 - Test
+- Wait Until Datetime - Test
+- Calculate Severity - Standard - Test
```

Comment:
What are all those tests?

Reply:
🤷 
Removed that.

Reply:
-.- https://github.com/demisto/etc/issues/39205

Reply:
@ssokolovich Please remove it